blueprint/api.apib

```diff
@@ -1690,6 +1689,50 @@ This endpoint allows you to check whether a username is valid (by running a vali
 
     + Attributes (Unprocessable Entity Response)
 
+#StripeCard
+
+This resource identifies the relationship between a StripeCustomer and a StripeCard. A StripeCustomer has many StripeCards and StripeCard belongs to a StripeCustomer.
+
+## StripeCards [/stripe_cards]
+
+### List all StripeCards [GET]
+
++ Request
+
+    + Headers
+
+            Accept: application/vnd.api+json
+            Authorization: Bearer {token}
+
++ Response 200 (application/vnd.api+json; charset=utf-8)
+
+    + Attributes (StripeCards Response)
+
+## StripeCard [/stripe_cards/{id}]
+
+### Retrieve a StripeCard [GET]
+
+    + Parameters
+
+        + id (number) - Id of StripeCard.
+        +stripe_customer_id(number) - ID of StripeCustomer.
+
+    + Request
+
+        + Headers
+
+                Accept: application/vnd.api+json
+                Authorization: Bearer {token}
+
+    + Response 200 (application/vnd.api+json; charset=utf-8)
+
+        + Attributes (StripeCard Response)
+
+    + Response 404 (application/vnd.api+json; charset=utf-8)
+
+        + Attributes (Record Not Found Response)
```

Comment:
I would also add in the `401` and `403` response since we know that this is authenticated and authorized.